<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagerWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Продолжить последнюю игру</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QTableWidget" name="tableWidget_2"/>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Включая бета-версии</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Все игры</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Только доступные</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Частично доступные</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Доступные и частично доступные</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Доступные и непроверенные</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Играть</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Обновить данные</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>&quot;</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>!</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
